src/helpers.js

```diff
@@ -51,6 +51,21 @@ module.exports = ({ types: t }) => {
   };
 
   const createRequireExpression = (dependencyNode, variableName) => {
```

Comment:
I'd like to rename this function since it's no longer just returning require expressions. How about something like `createDependencyInjectionExpression`? Let me know if you think of something better.

Reply:
Ok, seems to be more accurate.